core/trino-main/src/main/java/io/trino/sql/gen/ExpressionProfiler.java

```diff
@@ -73,4 +73,9 @@ public boolean isExpressionExpensive()
     {
         return isExpressionExpensive;
     }
+
+    public double getTotalExecutionTimeNanos()
```

Comment:
Why double?

Reply:
That's because it is stored as a `double` in this class. I've updated the code to remove this and get the time taken as a long in a slightly different way now.